AppModelv2-WebApp-OpenIDConnect-DotNet/Startup.cs

```diff
@@ -48,6 +48,13 @@ public void Configuration(IAppBuilder app)
                 Scope = OpenIdConnectScope.OpenIdProfile,
                 // ResponseType is set to request the code id_token - which contains basic information about the signed-in user
                 ResponseType = OpenIdConnectResponseType.CodeIdToken,
+                // ValidateIssuer set to false to allow personal and work accounts from any organization to sign in to your application
```

Comment:
This is not the right fix
you'r want to use the [AadIsserValidator](https://learn.microsoft.com/en-us/dotnet/api/microsoft.identitymodel.validators.aadissuervalidator?view=msal-web-dotnet-latest)

Reply:
The reason that I added this PR is because the code in this repo is used as a code sample in the quickstart of an app registration in Azure.
The code being downloaded for the user is unusable without these lines of code although the instructions at the bottom of the quickstart page include them.
A user looked for help using a quickstart page turned to a forum I'm a member in and had the following error
_Error: IDX10205: Issuer validation failed. Issuer: 'System.String'. Did not match: validationParameters.ValidIssuer: 'System.String' or validationParameters.ValidIssuers: 'System.String'_
The code downloaded in the sample should run without any change and the instructions at the bottom should explain what needs to be changed to make it more secure.
Or maybe... add the URL that was added in step 1 of the quick start to the app registration to the trusted issuers in the code as an injection once the repo was downloaded?

Thanks for the suggestion but I just copied the code from the instructions of the quickstart, do you mean the quickstart instructions are not correct?